packages/graphql-mocks/src/resolver-map/embed.ts

```diff
@@ -77,15 +78,21 @@ export function embed({
         );
       }
 
-      const wrappedResolver = await applyWrappers(resolverToEmbed, [...wrappers, embedPackOptionsWrapper], {
-        type,
-        field,
-        schema,
-        resolverMap,
-        packOptions,
-      });
+      if (wrappers?.length) {
+        // if there is at least one wrapper, the embedPackOptionsWrapper should also
+        // be included in order to better manage the context and ensure that the `pack`
+        // property is included
+        wrappers = [embedPackOptionsWrapper, ...wrappers];
```

Comment:
Set `embedPackOptionsWrapper` before any other wrappers so that it can ensure `pack` is included on the context in case wrappers after it drop it somehow.